### PR TITLE
Fix Java logging in gRPC Attestation

### PR DESCRIPTION
**Pull request by ipetr0v**
Fixes https://github.com/project-oak/oak/issues/2136

### CODE REVIEW

File: oak_functions/client/java/src/AttestationClient.java [hunk 1]
```diff
@@ -43,14 +40,14 @@ public void onNext(AttestedInvokeResponse response) {
                 try {
                     messageQueue.put(response);
                 } catch (Exception e) {
-                    logger.log(Level.WARNING, "Couldn't send server response to the message queue: {0}", e);
+                    System.out.printf("Couldn't send server response to the message queue: %s\n", e.toString());
```
Comment (rbehjati) [inline oak_functions/client/java/src/AttestationClient.java:43]: Is the call to `toString` really needed?

**Review by rbehjati — APPROVED** [review]

File: oak_functions/client/java/src/AttestationClient.java (hunk 1 above)
Comment (ipetr0v) [reply oak_functions/client/java/src/AttestationClient.java:43]: Yes, in order to print it via `%s`.
IIUC Java cannot print raw Objects using `System.out.printf` (but please correct me if I'm wrong)

Comment (rbehjati) [reply oak_functions/client/java/src/AttestationClient.java:43]: I could answer this question a year ago. My java is very rusty now :D

**Comment by tiziano88** [comment]
I don't think we should remove proper logging support, especially since this library is going to be used in Android and other places which expect canonical logging formats. 

Definitely it should not pritn to stdout anyways, at most to stderr.

**Comment by ipetr0v** [comment]
Updated and returned `Logger` back.

**Comment by tiziano88** [comment]
Why did the hashes for oak_loader and oak_functions_loader change?

**Comment by conradgrobler** [comment]
> Why did the hashes for oak_loader and oak_functions_loader change?

There were two earlier dependabot updates where the workflow to update these failed. This was the first successful run on `main` since then: https://github.com/project-oak/oak/actions/workflows/reproducibility.yaml

**Pull request merged**